test/e2e/system_connection_test.go

```diff
@@ -253,6 +253,16 @@ var _ = Describe("podman system connection", func() {
 			u, err := user.Current()
 			Expect(err).ShouldNot(HaveOccurred())
 
+			// Ensure that the remote end uses our built podman
+			if os.Getenv("PODMAN_BINARY") == "" {
+				err = os.Setenv("PODMAN_BINARY", podmanTest.PodmanBinary)
+				Expect(err).ShouldNot(HaveOccurred())
+
+				defer func() {
+					os.Unsetenv("PODMAN_BINARY")
+				}()
```

Comment:
I saw a oneliner `defer` earlier today, so much nicer. Can I (and should I) rewrite that as follows?
```go
    defer os.Unsetenv("PODMAN_BINARY")
```

Reply:
Yes but I don't think we need to do that now, do it in next cleanup.